pkg/prowgen/prowgen_test.go

```diff
@@ -242,10 +257,19 @@ func TestGeneratePostSubmitForTest(t *testing.T) {
 				Branch: "Branch",
 			}},
 		},
+		{
+			name: "fourth",
+			repoInfo: &ProwgenInfo{Metadata: ciop.Metadata{
+				Org:    "organization",
+				Repo:   "repository",
+				Branch: "branch",
+			}},
+			jobRelease: "4.6",
```

Comment:
Do we have a higher level test that would validate that we're reading the correct field in the ci-op config to populate this?

Reply:
I don't think we have anything already. I would expect something like that to be in config/load_test.go
I couldn't find any existing tests for the rest of the fields, am I looking in the wrong place or did you mean that I should write the test for all the fields?

Reply:
We have integration tests for this binary -- please add a test to that one

Reply:
Alright, I added an integration test to test for the job-release label, I have done it only for the periodic though.